client-spark/common/src/main/java/org/apache/uniffle/shuffle/manager/RssShuffleManagerBase.java

```diff
@@ -947,23 +947,23 @@ protected Map<Integer, List<ShuffleServerInfo>> requestShuffleAssignment(
     int retryTimes = sparkConf.get(RssSparkConfig.RSS_CLIENT_ASSIGNMENT_RETRY_TIMES);
     faultyServerIds.addAll(rssStageResubmitManager.getServerIdBlackList());
     try {
-      ShuffleAssignmentsInfo response =
-          shuffleWriteClient.getShuffleAssignments(
-              appId,
-              shuffleId,
-              partitionNum,
-              partitionNumPerRange,
-              assignmentTags,
-              assignmentShuffleServerNumber,
-              estimateTaskConcurrency,
-              faultyServerIds,
-              stageId,
-              stageAttemptNumber,
-              reassign,
-              retryInterval,
-              retryTimes);
       return RetryUtils.retry(
           () -> {
+            ShuffleAssignmentsInfo response =
```

Comment:
How about add a simple comment here to explain why the retryInterval and retryTimes are `0` here, so that reviewer and other developer can understand the motivation of this change.

Reply:
done

Reply:
done, modified the exception message